.github/workflows/deploy.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ 'develop', 'master' ]
+  pull_request:
+    branches: [ 'develop' ]
```

Comment:
Why just develop and not master as well here?

Reply:
Ok, that's done.